### ISSUE TITLE
Function DiscardVirtualMemory not found in kernel33.dll when trying to run Lunar Client.exe

### ISSUE THREAD

**Issue by MasterMando0** [vxiiduu/VxKex#102]
Hello there! I cannot launch Lunar Client 3.1.3 on VxKex, also could you support more programs and make the extended kernel like Windows Vista's? Thank you.

**Comment by MacncheeseTTR** [comment]
Haha! i just tried to run lunar client yesterday on my machine. Unfortunately its not looking like they're gonna implement this or even continue the project, as there hasn't been a commit since january. I do hope they add something like this in the future. Im still trying to get a old launcher of lunar client working, so if i get that working ill let you know!

**Comment by MasterMando0** [comment]
> Haha! i just tried to run lunar client yesterday on my machine. Unfortunately its not looking like they're gonna implement this or even continue the project, as there hasn't been a commit since january. I do hope they add something like this in the future. Im still trying to get a old launcher of lunar client working, so if i get that working ill let you know!

Haha, I'm glad that I am not the only one that had this situation! Yeah, you may be right, thanks for letting me know if you can try get it working in the future :D - Best wishes :)

**Comment by MacncheeseTTR** [comment]
UPDATE: my friend is in the works of making a lunar client tui launcher, that doesnt even use the launcher, he has to get around microsoft logins, so it might take a while.

**Comment by MasterMando0** [comment]
So, does this mean the launcher is going to be for cracked minecraft or not?

**Comment by MasterMando0** [comment]
I'm using Crystal Client Launcher for Windows 7 (which is similar to lunar client but less custom) if u cannot fix lunar client

**Comment by MacncheeseTTR** [comment]
> So, does this mean the launcher is going to be for cracked minecraft or not?

No, it just means it might have to copy a Microsoft verification link to your clipboard and then you login to Microsoft, it reads the token, etc. It will just be a inconvenience.

**Comment by MasterMando0** [comment]
Oh cool.

**Comment by MacncheeseTTR** [comment]
> Oh cool. What is your PC specs on Windows 7?

Ryzen 7 3700x, rtx 3060. 16gb of ram

**Comment by MasterMando0** [comment]
Damn, that's high specs u got there. Originally I was trying to get an RTX 3050 driver for Windows 7 running on a ryzen 9 5900hx with 16gb and 1tb nvme ssd but the drivers for the 3050 does not work for some unknown reason even though i installed on the i chose the driver for windows 7. (i installed on pure uefi class 3 with no csm even though i did not have csm or legacy boot in my bios) do you know how i can solve this because i had to revert back to windows 10 and now i also have to use it and my other lenovo thinkpad t470p which has windows 7 as well, cheers if u can fix this issue.

**Comment by MacncheeseTTR** [comment]
for some reason the 30 series drivers are really weird, and you need to install a very specific driver. lemme find the original forum with the download link so that im not one of those people that are like "here, download this random link" lol

**Comment by MasterMando0** [comment]
alright lol i appreciate that thank you, how's the lunar client project going on?

**Comment by MasterSlenderTR** [comment]
https://drive.google.com/file/d/1tdEVKfEpaL50Fny2gRWy0rNFOVkianvz/view?usp=drive_link

**Comment by MasterMando0** [comment]
The link I downloaded gave me the same error. 
![image](https://github.com/vxiiduu/VxKex/assets/153443260/9c6c29f5-e804-4f7e-a2bf-a465d5ccde2a)

**Comment by MacncheeseTTR** [comment]
> The link I downloaded gave me the same error.

could be a virus, never know, but it works for me! go into resources then app-update.yml and delete everything, go into permissions and edit the permissions to deny for every user and admin, it should work now!

**Comment by MasterMando0** [comment]
Where do i need to go to permissions?

**Comment by MasterMando0** [comment]
oh nevermind i got it bro, thanks so much u legend, even u masterslender

**Comment by cutiegin** [comment]
https://github.com/win32ss/supermium-electron/

**Comment by vxiiduu** [comment]
Try it again, the new release of vxkex now supports electron apps.

**Comment by cutiegin** [comment]
> Try it again, the new release of vxkex now supports electron apps.

![image](https://github.com/vxiiduu/VxKex/assets/115403340/b87f8d91-39fc-41a7-a7ff-b82d650ff319)

**Comment by vxiiduu** [comment]
This issue is caused by the lack of GetHostNameW from ws2_32.dll - I will add it in the next release. Similar to the one where someone couldn't get VSCode running.

**Comment by MasterMando0** [comment]
Hey vxiiduu, does this mean you will commit working on VxKex? That would be
amazing if you planned on more and more of future releases :)

**Comment by vxiiduu** [comment]
Yes, I've started working on VxKex again now.

**Comment by MasterMando0** [comment]
That's good, also how long will future releases take?

**Comment by vxiiduu** [comment]
Things will be released when they're ready. I can't promise an exact date, but I can tell you that I've already gotten VSCode and Signal working, which means this lunar client thing is probably going to work too (since they all depend on GetHostNameW). Definitely within the next 2 weeks, probably sooner.

**Comment by MasterMando0** [comment]
That's good. :)
